mlflow/tracking/fluent.py

```diff
@@ -955,6 +940,7 @@ def search_runs(
     run_view_type=ViewType.ACTIVE_ONLY,
     max_results=SEARCH_MAX_RESULTS_PANDAS,
     order_by=None,
+    as_pandas=True,
```

Comment:
Nit: I think it would be clearer / more pythonic / more extensible to use `format="pandas"` and `format="list"` rather than `as_pandas=True/False`. Let me know what you think here - sorry for the hassle!

Reply:
format is a builtin funciton in python. I am changing to format but I usually avoid overriding builtins

Reply:
I thought of output_format but it feels too verbose

Reply:
I am not sure conditional outputs is worth making extensible

tests/statsmodels/model_fixtures.py:    data = sm.datasets.scotland.load(as_pandas=False)

We won't be the only library making this distinction either so it might be ok to keep it rather than allow for that extensibility

xgboost, statsmodels, and autogluon also uses as_pandas

alphavantage and iexfinance uses output_format='pandas' probably for the builtin issue

I think given statsmodels and xgboost are also flavors it might be ok to mirror their approach with as_pandas unless we have intentions to add more formats